### ISSUE TITLE
Bug: shell and indexquestion do not work on nixos

### ISSUE THREAD

**Issue by tompflynn** [bakks/butterfish#7]
here is an example of running butterfish shell. Occasionally words like "status" or "--help" will pop up in gray, and you cannot exit the program without closing the terminal.
![image](https://user-images.githubusercontent.com/129278154/236963096-7d4023da-b04f-42c9-abdc-e954a0766184.png)
The command was run on alacritty with bash, but the same thing happens on wezterm or using other shells.

here is an example of running butterfish indexquestion "test"
![image](https://user-images.githubusercontent.com/129278154/236963421-7e29badb-cb00-494c-b8fc-67868dbb1bf2.png)
all these commands somewhat work:
```
butterfish index
butterfish indexsearch
butterfish loadindex
```
but butterfish indexquestion does not

**Comment by bakks** [comment]
Hey @tompflynn - thanks for creating an issue, sorry this isn't working as expected. I'm going to try to reproduce the error right now.

One theory I have about the shell problem is that [this code added recently](https://github.com/bakks/butterfish/blob/main/butterfish/shell.go#L544) is causing the problem. If that's the case a workaround would be to try running `butterfish shell -p''` to see if that makes a difference.

Something that would be helpful - what's in your `.bashprofile` or `.profile`? I haven't seen anything like this on a handful of different shells so I'm wondering if there's some more custom/esoteric setup happening.

**Comment by bakks** [comment]
Spun up a NixOS, wasn't able to replicate. My best guess is there's something a bit custom going on in a file that bash is loading like `.bashprofile` or `.profile`.

**Comment by bhazzard** [comment]
To show what the ux seems to be on Mac using zsh.

`$ butterfish loadindex .`

```
Loading indexes (not generating new embeddings) for .
Loaded 56 files
```

`butterfish indexquestion "is this working?"`

```
Error: No vector index loaded
```

**Comment by bakks** [comment]
Ok this should be fixed now in the v0.1.4 release - I definitely misunderstood the issue initially, thanks for investigating @bhazzard
